style.css

```diff
@@ -135,11 +144,225 @@ body {
   opacity: .5;
 }
 
-.main__image {
-  width: 770px;
-  height: 657px;
-}
-
 .main__title-text_highlight {
   background: #ecf3ff;
 }
+
+/* Мобильные устройства */
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .logo {
+    right: 12px;
+  }
+
+  .logo__circle {
+    transform: translateY(-95%);
+  }
+
+  .logo__circle_color_blue {
+    top: 6px;
+  }
+
+  .logo__text {
+    display: none;
+  }
+
+  .header {
+    margin-top: 51px;
+    margin-bottom: 33px;
+  }
+
+  .menu__item::after {
+    visibility: hidden;
+  }
+
+  .menu__item_active::after {
+    visibility: hidden;
+  }
+
+  .main {
+    text-align: center;
+    padding: 80% 0 0;
+    background-size: 95%;
+    background-position: top center;
+  }
+
+  .main__title {
+    margin: 0 0 33px 6px;
+    padding-top: 25px;
+    font: bold 30px/39px "pt_serif", arial, helvetica, sans-serif;
+    text-align: center;
+  }
+
+  .main__subtitle {
+    margin: 0 25px;
+    padding-bottom: 25px;
+    font: 500 16px/19px "Roboto", arial, helvetica, sans-serif;
+    text-align: center;
+  }
+
+  .main__button {
+    padding: 14px 22px;
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+  }
+
+/* Бургер меню */
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6297664)